### ISSUE TITLE
Stampede-Resistant Config Cache

### ISSUE THREAD

**Issue by seansan** [OpenMage/magento-lts#422]
@colinmollenhour 
nice work
is this something to add to LTS? (Or was it added before?)

https://gist.github.com/colinmollenhour/7a91c4a92ccfd2adaeb6

**Comment by LeeSaferite** [comment]
Perhaps we should add this to https://github.com/OpenMage/magento-lts/projects/3

**Comment by colinmollenhour** [comment]
Yes, I think it should be added as it definitely fixes some significant stability issues, although others have since forked my gist and maintained the patch more carefully so those may be a better starting point than the original. E.g.: https://github.com/magehost/Cm_StampedeResistantConfig

**Comment by seansan** [comment]
​good to hear; maybe give them a nudge? maybe they are willing to
contribute ... ​

**Comment by sreichel** [comment]
> Perhaps we should add this to Project Cleanup

Yep. Same for #267 or #374

**Comment by seansan** [comment]
@colinmollenhour Is this still something to add? Or have you/we changed our views on this?
